src/howitz/endpoints.py

```diff
@@ -475,6 +475,49 @@ def show_update_events_status_modal():
     return render_template('/components/popups/modals/update-event-status-modal.html', current_state='open')
 
 
+@main.route('/event/<i>/poll', methods=["POST"])
+def poll(i):
+    selected_events = session.get("selected_events", [])
+    event_id = int(i)
```

Comment:
I'm starting to think we should have a polish-issue fixing up all the url-arguments...